cfgov/v1/management/commands/delete_page_cache.py

```diff
@@ -4,15 +4,16 @@
 
 
 class Command(BaseCommand):
-    help = "Delete the cache of a page by its full URL"
+    help = "Delete the cache of pages by full URLs"
 
     def add_arguments(self, parser):
         parser.add_argument(
             "--url",
             required=True,
-            help="The full URL for the page you wish to delete its cache",
+            help="The full URLs for pages you wish to delete its cache",
         )
 
     def handle(self, *args, **options):
-        page_url = options["url"]
-        purge_url_from_cache(page_url)
+        page_urls = options["url"]
+        for page_url in page_urls.split(" "):
+            purge_url_from_cache(page_url)
```

Comment:
With `nargs="+"` suggestion, this can be simplified:

```suggestion
        purge_urls_from_cache(options["url"])
```

(Also requires changing `import` statement at the top of this file.)

Reply:
Used `purge_urls_from_cache` and `nargs` to simplify.